### PR TITLE
add beacon

### PR DESCRIPTION
**Pull request by MalinAhlberg**
Adds a beacon with data from ejprd-data. Follow instructions in `docs/beacon.md`:

1. (First time only:) Run `git submodule init`
1. Run `git submodule update --remote` to checkout the beaconized data
1. Start beacon: `docker compose up`
1. (First time only: )Run: `config/beacon_db_load.sh` to load all data to the db
1. Beacon should be running at port 5050, try it `curl localhost:5050/api | jq .`
1. Try to search for a specific variant:
```
curl 'https://beacon.gdi.elixir-greece.org/api/g_variants?referenceName=22&start=16050114&end=16050115&referenceBases=A&alternateBases=G' | jq .
```

**Future work**:
connecting to lsaai. This instance will only work with the datasets from `ejprd-data`. No authorization is done.

### CODE REVIEW

File: config/gdi-starter-kit/docker-compose.override.yml
Comment (wna-se) [inline config/gdi-starter-kit/docker-compose.override.yml:1]: Perhaps the base configuration of the `beacon` and `db` should be moved to the docker-compose.yml file and then just leave the port and network mappings for the services in this file to be consistent with the other services.

File: config/gdi-starter-kit/docker-compose.override.yml [hunk 1]
```diff
@@ -103,6 +103,40 @@ services:
...
+    ports:
+      - 27019:27017
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: root
+      MONGO_INITDB_ROOT_PASSWORD: example
+      MONGO_INITDB_DATABASE: beacon
+    volumes:
+      - ./mongo-init/:/docker-entrypoint-initdb.d/:ro
+      - ../../data/beaconized/:/docker-entrypoint-initdb.d/:ro
+      - ./config/beacondb/:/data/db/
```
Comment (wna-se) [inline config/gdi-starter-kit/docker-compose.override.yml:136]: I would prefer this to be a temporary volume in docker. E.g., add a named volume below and reference it here.

File: docs/beacon.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Beacon v2
+
+1. (First time only:) Run `git submodule init`
+1. Run `git submodule update --remote` to checkout the beaconized data
+1. Start beacon: `docker compose up`
+1. (First time only: )Run: `config/beacon_db_load.sh` to load all data to the db
+1. Beacon should be running at port 5050, try it `curl localhost:5050/api | jq .`
```
Comment (wna-se) [inline docs/beacon.md:7]: What should I look for in the response to know that everything is working?

Comment (MalinAhlberg) [reply docs/beacon.md:7]: The `response.name` should be "EJPRD Swedish Test Beacon".
The info shown here can be configured from `config/beacon_conf.py`, please update as you see fit.

Comment (wna-se) [reply docs/beacon.md:7]: That seems to work but the datasets key is an empty array. Is there a way to make sure that the datasets have been properly indexed? Is there a repository with the Docker image?

Comment (MalinAhlberg) [reply docs/beacon.md:7]: I'm not sure why the `datasets` is empty, but it seems to be the same for other beacons, even if they have public data. 

However, there is probably a bug in our current db set up, will push a fix soon.

**Pull request merged**